From 4a8a46aba6b9e39bfb02c7f46933079b5a50eff5 Mon Sep 17 00:00:00 2001
From: Akseli Lahtinen <akselmo@akselmo.dev>
Date: Wed, 19 Nov 2025 11:49:40 +0200
Subject: [PATCH] Menu: Set ItemSpacing to 2

This had been forgotten to change in
https://invent.kde.org/plasma/breeze/-/merge_requests/563

This is the same spacing as QtQuick styles use
---
 kstyle/breezemetrics.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kstyle/breezemetrics.h b/kstyle/breezemetrics.h
index efad51c3c..647929013 100644
--- a/kstyle/breezemetrics.h
+++ b/kstyle/breezemetrics.h
@@ -55,7 +55,7 @@ struct Metrics {
     static constexpr int MenuItem_HighlightGap = 4;
     static constexpr int MenuItem_ExtraLeftMargin = 4;
     static constexpr int MenuItem_MarginHeight = 5;
-    static constexpr int MenuItem_ItemSpacing = 4;
+    static constexpr int MenuItem_ItemSpacing = 2;
     static constexpr int MenuItem_AcceleratorSpace = 16;
     static constexpr int MenuItem_TextLeftMargin = 8;
 
-- 
GitLab

